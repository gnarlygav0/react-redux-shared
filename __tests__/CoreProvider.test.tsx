import React from 'react'
import {create} from 'react-test-renderer'
import {CoreProvider} from '..'

const props = {
  data: {prop: 3},
  configuredStore: {
    state: {},
    getState: jest.fn(),
    subscribe: jest.fn(),
    dispatch: jest.fn(),
    replaceReducer: jest.fn(),
  },
} as any

describe('core provider', () => {
  it('works with configure store', () => {
    const component = create(
      <CoreProvider {...props}>
        <></>
      </CoreProvider>,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
  it('works with OUT configure store', () => {
    const component = create(
      <CoreProvider>
        <></>
      </CoreProvider>,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
