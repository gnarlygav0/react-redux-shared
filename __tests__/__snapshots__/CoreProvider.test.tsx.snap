// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core provider works with OUT configure store 1`] = `null`;

exports[`core provider works with configure store 1`] = `null`;
